### ISSUE TITLE
Search results

### ISSUE THREAD

**Issue by CcMarc** [numinix/ajax_auto_search_suggest_public#1]
The search results are not finding the actual product when it exist. For example I have a product called Hazelnut and when I search for that it brings up Banana Hazelnut and many others with the Hazelnut in the name but not actual Hazelnut as the first item?

**Comment by RobNuminix** [comment]
The module should be returning your exact search results just like the
advanced search result page.  To test, disable the module and perform a
search and they should be the same.  Or, click the search button and the
search results page will load.  This plugin only saves the user time by
suggesting various search terms from the search results.

To improve the relevance of search, I recommend a plugin called Zen Cart
Search Reloaded.  You will need your host to add FULL TEXT SEARCH in MySQL
and run a database repair once the plugin is installed.

On Thu, Jul 16, 2015 at 11:18 AM, CcMarc <notifications@github.com> wrote:

> The search results are not finding the actual product when it exist. For
> example I have a product called Hazelnut and when I search for that it
> brings up Banana Hazelnut and many others with the Hazelnut in the name but
> not actual Hazelnut as the first item?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numinix/ajax_auto_search_suggest_public/issues/1>.
>



-- 

 *Jeff Lew*
President

*Phone:* 1-855-686-4649 Ext 700
*Email/Gchat:* jlew@numinix.com
*Skype:* numinix
 [image: Numinix] <http://www.numinix.com/>

 [image: Facebook] <http://www.facebook.com/numinix> [image: Twitter]
<http://twitter.com/numinix> [image: Google+]
<https://plus.google.com/+Numinix> [image: Instagram]
<http://instagram.com/numinixwebdev> [image: Pinterest]
<http://www.pinterest.com/numinix> [image: Blog]
<http://www.numinix.com/blog/?feed=rss2>

**Comment by CcMarc** [comment]
Ok I will try and checkout the other mod FYI I was using a FREE module called InstantSearch and it worked great and if I typed Hazelnut it would go directly to that product but did not have product pictures as you typed in the search box that is why I switched to yours. You should look at that one and try to replicate it because the the search results were good.

**Comment by RobNuminix** [comment]
Yes, we may want to do that.  We did it this way in case there were other
products you wanted to show as well.  The reloaded search is still a great
idea.

On Thu, Jul 16, 2015 at 4:04 PM, CcMarc <notifications@github.com> wrote:

> Ok I will try and checkout the other mod FYI I was using a FREE module
> called InstantSearch and it worked great and if I typed Hazelnut it would
> go directly to that product but did not have product pictures as you typed
> in the search box that is why I switched to yours. You should look at that
> one and try to replicate it because the the search results were good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numinix/ajax_auto_search_suggest_public/issues/1#issuecomment-122126544>
> .
>



-- 

 *Jeff Lew*
President

*Phone:* 1-855-686-4649 Ext 700
*Email/Gchat:* jlew@numinix.com
*Skype:* numinix
 [image: Numinix] <http://www.numinix.com/>

 [image: Facebook] <http://www.facebook.com/numinix> [image: Twitter]
<http://twitter.com/numinix> [image: Google+]
<https://plus.google.com/+Numinix> [image: Instagram]
<http://instagram.com/numinixwebdev> [image: Pinterest]
<http://www.pinterest.com/numinix> [image: Blog]
<http://www.numinix.com/blog/?feed=rss2>
